src/app/organization/generate/generate.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="card">
+  <form enctype="multipart/form-data" method="POST">
+
```

Comment:
method is not required, you may use template driven or reactive form feature of angular. They are very powerful form feature.